### ISSUE TITLE
iframe.contentDocument is null in firefox 66

### ISSUE THREAD

**Issue by Dema** [storybooks/storybook#6211]
**Describe the bug**
Crashes in Firefox 66 on this line:
https://github.com/storybooks/storybook/blob/e14b5c5f6dd50a5875f822627e23e356807a97a6/lib/components/src/tooltip/WithTooltip.js#L110

`iframe.contentDocument == null` here
iframe is from  an extension https://github.com/wolfmanstout/vimium
![image](https://user-images.githubusercontent.com/207726/54741139-82201e80-4bce-11e9-92bb-86e832f54daf.png)

**To Reproduce**
1. start storybook
2. Wait for it to open
3. Blank screen with error `TypeError: iframe.contentDocument is null`

**Expected behavior**
Storybook UI opened

**System:**
 - OS: Arch Linux
 - Device: Desktop x64
 - Browser: Firefox 66.0
 - Framework: React
 - Addons: 
    "@storybook/addon-actions": "^5.0.3",
    "@storybook/addon-info": "^5.0.3",
    "@storybook/addon-links": "^5.0.3",
    "@storybook/addon-viewport": "^5.0.3",
 - Version: 5.0.3

**Additional context**

**Comment by Armanio** [comment]
Give me 15 minutes!

**Comment by shilman** [comment]
Son of a gun!! I just released https://github.com/storybooks/storybook/releases/tag/v5.1.0-alpha.13 containing PR #6238 that references this issue. Upgrade today to try it out!

Because it's a pre-release you can find it on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.

**Comment by shilman** [comment]
Yay!! I just released https://github.com/storybooks/storybook/releases/tag/v5.0.4 containing PR #6238 that references this issue. Upgrade today to try it out!

**Comment by Dema** [comment]
There's something strange happened with releases. `yarn upgrade-interactive --latest` wants to install 3.4.12 instead of 5.0.4.
on  https://www.npmjs.com/package/@storybook/react it says that 
```
Current Tags

    4.0.0-alpha.25    alpha
    3.4.12    latest
    4.0.0-rc.6    rc
    5.1.0-alpha.13    next
    5.0.0-debug.3    debug
```

**Comment by shilman** [comment]
Thanks @Dema for letting me know. I did new releases for `3.4.x` and `4.1.x` to workaround a bug in one of Storybook's dependencies, but I didn't realize that would mess up the `latest` NPM tag. I just released `5.0.5` to fix it. https://github.com/storybooks/storybook/releases/tag/v5.0.5
